.eslintignore

```diff
@@ -6,6 +6,7 @@ vendor/
 /packages/-ember-data/docs/
 dist
 tmp
+.git/
```

Comment:
O_o I've never seen eslint try to traverse the .git directory before!?!

Reply:
but it does :( I was digging into our eslint perf and this was one of a few areas causing weirdness.